pkg/controller/migmigration/directimagemigration.go

```diff
@@ -88,3 +111,34 @@ func (t *Task) setDirectImageMigrationWarning(dim *migapi.DirectImageMigration)
 		})
 	}
 }
+
+func (t *Task) deleteDirectImageMigrationResources() error {
+
+	// fetch the DIM
+	dim, err := t.getDirectImageMigration()
+	if err != nil {
+		return liberr.Wrap(err)
+	}
+
+	if dim != nil {
+		// fetch the corresponding DISM instance
+		dism, err := t.getDirectImageStreamMigrationForDIM(dim)
```

Comment:
As above, this should return a list of DISM resources rather than just one.

Reply:
Done